fdbclient/include/fdbclient/TenantSpecialKeys.actor.h

```diff
@@ -403,6 +403,7 @@ class TenantRangeImpl : public SpecialKeyRangeRWImpl {
 	}
 
 	Future<Optional<std::string>> commit(ReadYourWritesTransaction* ryw) override { return commitImpl(this, ryw); }
+	bool supportsTenants() const override { return true; };
```

Comment:
I think this one should remain false; we won't support modifying or reading tenant state inside of a tenant transaction.